homework/src/App.js

```diff
@@ -5,77 +5,72 @@
 class App {
   constructor(url) {
     this.initialize(url);
+    this.root = document.getElementById('root');
+    this.header = Util.createAndAppend('header', this.root);
+    this.label = Util.createAndAppend('label', this.header, { text: 'HYF Repositories' });
+    this.select = Util.createAndAppend('select', this.header);
+    this.article = Util.createAndAppend('article', this.root);
+    this.leftDiv = Util.createAndAppend('div', this.article, { id: 'leftDiv' });
+    this.rightDiv = Util.createAndAppend('div', this.article, { id: 'rightDiv' });
   }
```

Comment:
Lines 8-14 look to me as 'initialization'. So why not move them to the `initialize()` method, e.g. before line 17. Note however that it is not an error to have these lines here in the constructor.